tests/main/snapd-reexec/task.yaml

```diff
@@ -38,6 +38,12 @@ execute: |
     mount --bind /tmp/old-info $SNAP_MOUNT_DIR/core/current/usr/lib/snapd/info
     systemctl start snapd.service snapd.socket
     snap list
+    for i in $(seq 60); do
```

Comment:
I think that a comment here and in `tests/main/catalog-update/task.yaml`, documenting why we need the loop would be helpful.